core/src/main/resources/oracle/weblogic/deploy/messages/wlsdeploy_rb.properties

```diff
@@ -1115,6 +1115,7 @@ WLSDPLY-09332=Application {0} not found, found similar names {1}.  For deleting
 WLSDPLY-09333=Library {0} not found. Please specify a valid library for deletion
 WLSDPLY-09334=Application {0} not found. Please specify a valid application for deletion
 WLSDPLY-09335=Undeploying {0} {1} from targets: {2}
+WLSDPLY-09336=Application {0} old and new are at the same location. Automatic redeploy of application.
```

Comment:
WLSDPLY-09336=Application {0} old and new are at the same location. Automatic redeploy of application.
This message isn't overly clear.
Maybe something like:
WLSDPLY-09336=Redeployed application {0} because the old and the new binary have the same path.